Add known issue to cloud changelog (#6448)
mattermost/docs: source/install/cloud-changelog.md

 - Fixed an issue where certain hashtags were not searchable when using database search.

### Known Issues

>  - Clicking on the **replies** button doesn't navigate to the thread when Collapsed Reply Threads is enabled. A workaround is to use the reply arrow in the post menu or to navigate to the Threads view when replying to messages [MM-52995](https://mattermost.atlassian.net/browse/MM-52995). 

Reviewer: ```suggestion
 - Selecting the **replies** option for a thread doesn't navigate to the thread when Collapsed Reply Threads is enabled. A workaround is to use the reply arrow in the post menu, or to navigate to the **Threads** view when replying to messages [MM-52995](https://mattermost.atlassian.net/browse/MM-52995). 
```
